src/main/java/edu/ucdavis/dss/ipa/api/components/assignment/views/AssignmentExcelView.java

```diff
@@ -64,16 +64,14 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
                 SectionGroup sectionGroup = this.getSectionGroupByCourseAndTermCode(course, state.getTermCode());
 
                 if (sectionGroup != null) {
-                    excelHeader.createCell(col).setCellValue(
-                            StringUtils.join(
-                                    sectionGroup.getTeachingAssignments()
-                                            .stream()
-                                            .filter(TeachingAssignment::isApproved)
-                                            .map(ta -> ta.getInstructor().getLastName() + " " + ta.getInstructor().getFirstName().charAt(0))
-                                            .collect(Collectors.toList())
-                                    , ", "
-                            )
-                    );
+                    String instructorNames = "";
+                    for (TeachingAssignment teachingAssignment : sectionGroup.getTeachingAssignments()) {
+                        String name = teachingAssignment.getInstructor() != null ? teachingAssignment.getInstructor().getLastName() + " " + teachingAssignment.getInstructor().getFirstName().charAt(0) : teachingAssignment.getInstructorType().getDescription();
```

Comment:
Establish a teachingAssignment.getInstructorDisplayName() (or whatever you call this) and move that logic there.